.travis.yml

```diff
@@ -13,7 +13,11 @@ install:
 script:
   - cd test
   - pip freeze  # so to help eventual debug: know what exact versions are in use can be rather useful.
-  - nosetests -xv --process-restartworker --processes=1 --process-timeout=999999999  --with-cov --cov=alignak
+  - which nosetests
+  - nosetests -V
```

Comment:
Do you want to keep those lines? IMO we sould put that in a script because it is basically 5 lines giving only info about the system